pkgs/development/python-modules/rstcheck/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, docutils
+}:
+
+buildPythonPackage rec {
+  pname = "rstcheck";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "92c4f79256a54270e0402ba16a2f92d0b3c15c8f4410cb9c57127067c215741f";
+  };
+
+  propagatedBuildInputs = [ docutils ];
+
```

Comment:
The upstream repo contains a [test](https://github.com/myint/rstcheck/blob/master/test_rstcheck.py). If possible then we should execute that test otherwise add `doCheck = false;`.

Add `pythonImportsCheck` as a safe-guard.

Reply:
The test doesn't seem to be in the PyPI tar. Added doCheck false and pythonImportsCheck